frontend/pages/policies/ManagePoliciesPage/components/InstallSoftwareModal/InstallSoftwareModal.tsx

```diff
@@ -231,8 +233,10 @@ const InstallSoftwareModal = ({
             )}
           </ul>
           <span className="form-field__help-text">
-            Selected software will be installed when hosts fail the chosen
-            policy.{" "}
+            Selected software will be installed when hosts fail the policy. Host
+            counts will reset when a new software is
+            <br />
```

Comment:
We want the line break here rather than letting it wrap naturally? 👀

Reply:
Well, that's as [it's specced](https://www.figma.com/design/G41082SprvqvhGg6ZXwwN4/%2317129-Policy-automations%3A-run-script?node-id=5426-7077&node-type=section&t=sC67hh35ZusQC822-11). I don't have a preference. @marko-lisica?

Reply:
Since it matches the Figma I'll go ahead and mark this ready for review, though we can easily change this at any point if we want.

Reply:
In Figma we have this link as a separate layer, so I assumed it would be naturally wrapped after "is". I'm ok if we force a break. It would be weird if only "learn more" link went into the next row.